sql/catalyst/src/main/scala/org/apache/spark/sql/QueryCompilationErrors.scala

```diff
@@ -31,7 +31,7 @@ import org.apache.spark.sql.types.{AbstractDataType, DataType, StructType}
 
 /**
  * Object for grouping all error messages of the query compilation.
- * Currently it includes all [[AnalysisException]]s.
+ * Currently it includes all AnalysisExceptions.
```

Comment:
@yaooqinn Thank you. But I don't know the reason. Could you tell me why?

Reply:
@yaooqinn, I think you had to try with fully qualified name like `[[org.apache.spark.sql.AnalysisException]]` because regular classes should be able to use in Unidoc, but not `trait`s. in general.

There are still a bit of buggy behaviours in Unidoc even for the regular classes. In this case, you can use ` ``AnalysisException`` `.

Reply:
Yes I had tested the full class name.

I used AnalysisException directly w/o `` because it was AnalysisException before it changed to [[AnalysisException]]. I also believe `AnalysisException` is better